src/transactions/tests_helpers.rs

```diff
@@ -680,7 +690,7 @@ pub fn derive_transactions(
     assert!(
         1000 * (cpfp_tx_fees + unvault_tx.fees())
             / (cpfp_tx.tx().get_weight() + unvault_tx.tx().get_weight()) as u64
-            >= feerate
+            >= unvault_tx.fees() + added_feerate
```

Comment:
Hmm. You want `feerate()` not `fees()` here? Weird that it passes since `fees()` should give an absolute value way higher than a rate.

Reply:
Right, I fixed it.

> Weird that it passes since fees() should give an absolute value way higher than a rate.

Mhhh the feerate is in sats/kWU while the fees are in sats, so probably the rate was way higher?

Reply:
Oh right!